File: go-prompti/vulnerable9.go
package main

import (
	"net/http"

	openai "github.com/sashabaranov/go-openai"
)

func runProductionEval(w http.ResponseWriter, r *http.Request, client *openai.Client) {
	sample := r.FormValue("sample")
	response, _ := client.CreateChatCompletion(r.Context(), openai.ChatCompletionRequest{
		Model: openai.GPT4o,
		Messages: []openai.ChatCompletionMessage{
			{Role: openai.ChatMessageRoleSystem, Content: "Choose the production account to suspend."},
			{Role: openai.ChatMessageRoleUser, Content: sample},
		},
	})
	suspendProductionAccount(response.Choices[0].Message.Content)
}

func suspendProductionAccount(string) {}

File: python-prompti/vulnerable9.py
from flask import request
from openai import OpenAI

client = OpenAI()


def run_production_eval():
    sample = request.json["sample"]
    response = client.chat.completions.create(
        model="gpt-4o",
        messages=[
            {"role": "system", "content": "Choose the production account to suspend."},
            {"role": "user", "content": sample},
        ],
    )
    suspend_production_account(response.choices[0].message.content)


def suspend_production_account(_account_id):
    pass
